src/main/java/bio/terra/service/auth/ras/EcmService.java

```diff
@@ -55,17 +60,50 @@ public PassportApi getPassportApi() {
     return new PassportApi(client);
   }
 
+  public void addRasIssuerAndType(RASv1Dot1VisaCriterion criterion) {
+    criterion.issuer(ecmConfiguration.getRasIssuer()).type(RAS_CRITERIA_TYPE);
+  }
+
   public ValidatePassportResult validatePassport(ValidatePassportRequest validatePassportRequest) {
     var passportApi = getPassportApi();
-    return passportApi.validatePassport(validatePassportRequest);
+    var result = passportApi.validatePassport(validatePassportRequest);
+
+    if (result.isValid()) {
+      var auditInfo = result.getAuditInfo();
+      var df = new SimpleDateFormat("MM-dd-yyyy HH:mm:ss z");
+      df.setTimeZone(TimeZone.getTimeZone("UTC"));
+      logger.info(
+          """
+              [Validate Passport Audit]:
+              NIH User ID: {},
+              Transaction Number: {},
+              Data Repository accessed: {},
+              Study/Data set accessed: {},
+              Date/Time of access: {}
+              """,
+          auditInfo.getOrDefault("external_user_id", "not found"),
+          auditInfo.getOrDefault("txn", "not found"),
+          "TDR",
```

Comment:
This should probably be a constant (probably the same some used to return the service name in our DRS controller (I think it's `ApplicationConfiguration.APPLICATION_NAME`)